src/app/core/components/left-nav/left-nav.component.ts

```diff
@@ -50,6 +51,7 @@ export class LeftNavComponent implements OnInit, OnDestroy {
       // If the content changed (different id), clear the selection (clear all tabs as we don't really know on which tab was the selection)
       if (prevContent?.type !== content?.type || prevContent?.route.id !== content?.route.id) {
         this.dataSources.forEach(l => l.removeSelection());
+        this.selectId.emit(content?.route.id);
```

Comment:
But so if content is undefined, you possibly emit "undefined" while you event emitter type does not allow undefined? 🤨 How does it compile ?
From what I see, that's a bug in angular (https://github.com/angular/angular/issues/35119)  which is supposed to be fixed. While not fixed, it will crash at runtime, so do the check!

Reply:
Fixed